docs/installing.md

```diff
@@ -32,6 +33,59 @@ By default kubtest is installed in `default` namespace but you can change it in
 
 If you want kubtest to provide the endpoint for the kubest dashboard use `kubectl kubtest install -i` with the `-i` or `--ingress` option, it will setup a ingress-nginx controller for you in a managed cluster(for baremetal clusters this should be set up manually before installing kubtest).
 
+## Kubtest's Dashboard Ingress Configuration
+
+Dashboard will bring you web-based UI for managing and seeing all the tests and its results via web-browser.
+### Enabling dashboard
+In order to enable dashbaord please provide Helm's set value as follow during installation:
+```
+helm install kubtest kubeshop/kubtest --set kubtest-dashboard.enabled="true"
+```
+By default it's disabled
+### configuration for the nginx-based ingress controller with the cert-manager pointed at Let'sencrypt. 
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    acme.cert-manager.io/http01-edit-in-place: "true"
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/ingress.global-static-ip-name: kubtest-demo
+    meta.helm.sh/release-name: kubtest-demo
+    meta.helm.sh/release-namespace: kubtest-demo
+    nginx.ingress.kubernetes.io/cors-allow-credentials: "false"
+    nginx.ingress.kubernetes.io/cors-allow-methods: GET
+    nginx.ingress.kubernetes.io/enable-cors: "true"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+  name: kubtest-dashboard-kubtest-demo
+  namespace: kubtest-demo
+spec:
+  rules:
+  - host: your.domain.name
+    http:
+      paths:
+      - backend:
+          service:
+            name: kubtest-dashboard
+            port:
+              number: 80
+        path: /
+        pathType: Prefix
+  tls:
+  - hosts:
+    - demo.kubtest.io
+    secretName: kubtest-demo-cert-secret # to be created by ytour certificate manager within k8s cluster.
+```
+If you don't need TLS enabled just omit TLS configuration part. 
+
+> Dashbaord talks to api-server via endpoint. Hence api-server will hvae to have DNS as well.
+
+Please note that you can install ingress for dashboard together with api-server ingress with the usage of Helm chart as well:
+```
+helm install kubtest kubeshop/kubtest --set kubtest-dashboard.enabled="true" --set kubtest-dashboard.ingress.enabled="true" --set api-server.ingress.enabled="true"
```

Comment:
We know that this setup will work on gcloud. What about other providers?

I would rather we hid this option because it might lead users to believe that it will work with any setup.
I'd rather go with some example how to set it up manually totally decoupled from kubtest

p.s. otherwise it looks good

Reply:
Will add a small comment as well. So that users could install it manually as well using our Helm chart.